src/Navigation.svelte

```diff
@@ -61,7 +61,7 @@
             </li>
             <li class="nav-item">
               <span class="nav-link">
-                <Link to="/users">{$_("nav.manage_users")}</Link>
+                <Link to="/users/profile">{$_("nav.userprofile")}</Link>
```

Comment:
Zarządzanie userami a informacje o profilu to chyba powinny być osobne podstrony? Tzn Przemek chciał moim zdaniem zachować to zarządzanie userami bo tam ich aktywuje / blokuje.